app/assets/stylesheets/ingredient_families.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the ingredient_families controller here.
```

Comment:
And these. We don't need them right now.

Reply:
Done.